docker/openemr/5.0.2/openemr.conf

```diff
@@ -1,40 +1,33 @@
 LoadModule rewrite_module modules/mod_rewrite.so
 LoadModule allowmethods_module modules/mod_allowmethods.so
 
-
-# The ServerName directive sets the request scheme, hostname and port that
-# the server uses to identify itself. This is used when creating
-# redirection URLs. In the context of virtual hosts, the ServerName
-# specifies what hostname must appear in the request's Host: header to
-# match this virtual host. For the default virtual host (this file) this
-# value is not decisive as it is used as a last resort host regardless.
-# However, you must set it for any further virtual host explicitly.
-#ServerName www.example.com
-
-
 ## Security Options
+# Strong HTTP Protocol
+HTTPProtocolOptions Strict
 Protocols http/1.1
-# Removes revealing HTTP Headers
+        
+# Don't Reveal Server
 ServerSignature off
 ServerTokens Prod
-TraceEnable Off
```

Comment:
@danehrlich1 , just checking why remove this line?

Reply:
Looked this up and see why you removed this line (apache docs state this is not a security vulnerability)